src/DependencyInjection/ConnecthollandUserExtension.php

```diff
@@ -44,21 +44,32 @@ public function prepend(ContainerBuilder $container)
         $config = $this->getResourceOwnersConfiguration($container);
         $container->prependExtensionConfig('hwi_oauth', $config);
 
-        if ($container->hasExtension('lexik_jwt_authentication')) {
-            $config = $this->getJwtConfiguration($container);
-            $container->prependExtensionConfig('lexik_jwt_authentication', $config);
-        }
+        $this->prependJwtConfiguration($container);
+        $this->prependApiPlatformConfiguration($container);
     }
 
-    private function getJwtConfiguration(ContainerBuilder $container): array
+    private function prependApiPlatformConfiguration(ContainerBuilder $container): void
     {
-        $config = [
-            'secret_key'  => $container->resolveEnvPlaceholders($container->getParameter('env(JWT_SECRET_KEY)'), true),
-            'public_key'  => $container->resolveEnvPlaceholders($container->getParameter('env(JWT_PUBLIC_KEY)'), true),
-            'pass_phrase' => $container->resolveEnvPlaceholders($container->getParameter('env(JWT_PASSPHRASE)'), false),
-        ];
+        if ($container->hasExtension('api_platform')) {
+            $config = [
+                'mapping'  => [
+                    'paths' => [sprintf('%s/../Message', __DIR__)],
+                ]
+            ];
+            $container->prependExtensionConfig('api_platform', $config);
+        }
+    }
 
-        return $config;
+    private function prependJwtConfiguration(ContainerBuilder $container): void
+    {
+        if ($container->hasExtension('lexik_jwt_authentication')) {
+            $config = [
+                'secret_key'  => '%kernel.project_dir%/'.$container->resolveEnvPlaceholders($container->getParameter('env(JWT_SECRET_KEY)'), true),
```

Comment:
I'd prefer to sprintf the env var value instead of concatenate. Double %-signs escapes

`sprintf('%%kernel.project_dir%%/%s', $container->resolveEnvPlaceholders($container->getParameter('env(JWT_SECRET_KEY)'), true)),`